### PR TITLE
Fix/partners

### PR DESCRIPTION
**Pull request by dimasster**
dev
## JIRA

* [Main JIRA ticket](https://jira.softserve.academy/secure/RapidBoard.jspa?rapidView=id)

## Code reviewers

- [ ] @github_username

### Second Level Review

- [ ] @github_username

## Summary of issue

ToDo

## Summary of change

ToDo

## Testing approach

ToDo

## CHECK LIST
- [ ]  СI passed
- [ ]  Сode coverage >=95%
- [ ]  PR is reviewed manually again (to make sure you have 100% ready code)
- [ ]  All reviewers agreed to merge the PR
- [ ]  I've checked new feature as logged in and logged out user if needed
- [ ]  PR meets all conventions

### CODE REVIEW

**Review by EyR1oN — APPROVED** [review]

File: src/features/StreetcodePage/PartnersBlock/PartnerItem/PartnerContent/PartnerContent.content.tsx [hunk 1]
```diff
@@ -30,16 +32,18 @@ const PartnerContent = ({ partner }:Props) => {
                     style={{ display: partner.partnerSourceLinks.length < 1 ? 'none' : '' }}
                 >
                     {partner.partnerSourceLinks.map((sl) => (
```
Comment (AleXLaeR) [inline src/features/StreetcodePage/PartnersBlock/PartnerItem/PartnerContent/PartnerContent.content.tsx:34]: `sl` parameter may be destructured to reduce boilerplate and increase readability.

File: src/features/StreetcodePage/PartnersBlock/PartnerItem/PartnerItem.styles.scss [hunk 2]
```diff
@@ -6,7 +6,7 @@
 .partnerItem {
     .partnerLogo {
         @include mut.sized($height: 100%);
-        filter: grayscale(90%);
+        filter: opacity(50%) grayscale(100%);
         transition: all .5ms ease;
```
Comment (AleXLaeR) [inline src/features/StreetcodePage/PartnersBlock/PartnerItem/PartnerItem.styles.scss:10]: - `transition` prop-ty should be vendored for cross-browser support.
- Also, a half of a millisecond? Is that even perceivable by a human eye?

File: src/features/StreetcodePage/PartnersBlock/PartnerItem/PartnerContent/PartnerContent.content.tsx [hunk 3]
```diff
@@ -2,13 +2,15 @@ import './PartnerContent.styles.scss';
...
 import Partner from '@/models/partners/partners.model';
 
 interface Props {
     partner: Partner
 }
 
-const PartnerContent = ({ partner }:Props) => {
+const PartnerContent = ({ partner }: Props) => {
     const { imagesStore } = useMobx();
     const { getImage } = imagesStore;
```
Comment (AleXLaeR) [inline src/features/StreetcodePage/PartnersBlock/PartnerItem/PartnerContent/PartnerContent.content.tsx:15]: These two declarations may be inlined.

File: src/features/StreetcodePage/PartnersBlock/PartnerItem/PartnerContent/PartnerContent.content.tsx [hunk 4]
```diff
@@ -2,13 +2,15 @@ import './PartnerContent.styles.scss';
 
 import useMobx from '@stores/root-store';
 
+import { Popover } from 'antd';
+
 import Partner from '@/models/partners/partners.model';
```
Comment (AleXLaeR) [inline src/features/StreetcodePage/PartnersBlock/PartnerItem/PartnerContent/PartnerContent.content.tsx:7]: Redundant slash between `@` and `models`.

File: src/features/StreetcodePage/PartnersBlock/PartnerItem/PartnerContent/PartnerContent.content.tsx [hunk 5]
```diff
@@ -2,13 +2,15 @@ import './PartnerContent.styles.scss';
...
 
 interface Props {
     partner: Partner
```
Comment (AleXLaeR) [inline src/features/StreetcodePage/PartnersBlock/PartnerItem/PartnerContent/PartnerContent.content.tsx:10]: Inconsistent style; no trailing semicolon.

File: src/features/StreetcodePage/PartnersBlock/PartnerItem/PartnerContent/PartnerContent.content.tsx [hunk 6]
```diff
@@ -30,16 +32,18 @@ const PartnerContent = ({ partner }:Props) => {
                     style={{ display: partner.partnerSourceLinks.length < 1 ? 'none' : '' }}
```
Comment (AleXLaeR) [inline src/features/StreetcodePage/PartnersBlock/PartnerItem/PartnerContent/PartnerContent.content.tsx:32]: `display` p-ty cannot be equal to an empty string.  The default disaplay of (almost) every html-tag is `block`.

File: src/features/StreetcodePage/PartnersBlock/PartnerItem/PartnerContent/PartnerContent.content.tsx [hunk 7]
```diff
@@ -2,13 +2,15 @@ import './PartnerContent.styles.scss';
...
+import { Popover } from 'antd';
+
 import Partner from '@/models/partners/partners.model';
 
 interface Props {
     partner: Partner
 }
 
-const PartnerContent = ({ partner }:Props) => {
+const PartnerContent = ({ partner }: Props) => {
```
Comment (AleXLaeR) [inline src/features/StreetcodePage/PartnersBlock/PartnerItem/PartnerContent/PartnerContent.content.tsx:13]: `partner` prop may be destructured.

File: src/features/StreetcodePage/PartnersBlock/PartnerItem/PartnerContent/PartnerContent.styles.scss [hunk 8]
```diff
@@ -12,6 +12,12 @@
 
 .header {
     @include mut.flexed($align-items: center, $gap: 14px);
+
+    p {
+        @include mut.with-font($font-family: f.$closer-text-font, $font-size: 20px, $font-weight: 500);
+        line-height: pxToRem(26.4px);
```
Comment (AleXLaeR) [inline src/features/StreetcodePage/PartnersBlock/PartnerItem/PartnerContent/PartnerContent.styles.scss:18]: `-line-height` p-ty shouldn't be set in **rems**.
Refer to: https://stackoverflow.com/questions/15132671/ems-for-line-height.
image

File: src/features/StreetcodePage/PartnersBlock/PartnerItem/PartnerContent/PartnerContent.styles.scss [hunk 9]
```diff
@@ -12,6 +12,12 @@
 
 .header {
     @include mut.flexed($align-items: center, $gap: 14px);
+
+    p {
+        @include mut.with-font($font-family: f.$closer-text-font, $font-size: 20px, $font-weight: 500);
```
Comment (AleXLaeR) [inline src/features/StreetcodePage/PartnersBlock/PartnerItem/PartnerContent/PartnerContent.styles.scss:17]: `font-size` is 20px by default.

File: src/features/StreetcodePage/PartnersBlock/PartnerItem/PartnerContent/PartnerContent.styles.scss [hunk 10]
```diff
@@ -29,11 +35,17 @@
 
     .mainLink {
         @include mut.truncated($line-num: 1);
+        @include mut.with-font($font-size: 20px, $font-weight: 300);
```
Comment (AleXLaeR) [inline src/features/StreetcodePage/PartnersBlock/PartnerItem/PartnerContent/PartnerContent.styles.scss:38]: Same as above.

**Review by AleXLaeR — CHANGES_REQUESTED** [review]

**Review by Tatiana2424 — APPROVED** [review]

**Review by AleXLaeR — APPROVED** [review]

**Pull request merged**